### DISCUSSION TITLE
Flink Doris Connector 是否支持2.0版本的doris

### DISCUSSION THREAD

**Discussion by ksdb1155228** [apache/doris#29217] [RFC]
我看官网提供的文档中，提到Flink Doris Connector的版本兼容表里，并没有包含2.0+版本。请问一下现有版本的Flink Doris Connector 是否已经支持doris 2.0+版本了。
![image](https://github.com/apache/doris/assets/57595395/f6d90f0d-baa0-46f0-94f8-45d0c12c10a3)

**Comment by LemonLiTree** [comment]
supported
